Gruntfile.js

```diff
@@ -480,10 +508,19 @@ module.exports = function( grunt ) {
 
 		// TODO fix issues, enable
 		// "jscs:dist",
-		"test:functional",
```

Comment:
There are two things I don't want to lose:

1. Having local tests run on the default task.
1. Having unit tests run before building (requirejs, ...) and having functional tests run before minification (uglify).